src/js/server/apply/router.js

```diff
@@ -275,4 +280,17 @@ function checkHasApplied(req, res, next) {
   }).catch(next);
 }
 
+/**
+ * Intercepts requests to check if applications are still open, redirecting to the dashbaord if not
+ */
+function checkApplicationsOpen(req, res, next) {
+  console.log(process.env.APPLICATIONS_OPEN);
+  if (process.env.APPLICATIONS_OPEN_STATUS === 'closed') {
```

Comment:
We check this env var in a bunch of places, sometimes referring to hard-coded strings and sometimes referring to constants. Is there a way we can make this more consistent and predictable?